src/client.rs

```diff
@@ -536,15 +595,33 @@ impl Spotify {
     /// - offset - the index of the first track to return
     /// - market - an ISO 3166-1 alpha-2 country code or the string from_token.
     ///
+    /// See [`Spotify::playlist_tracks`] for a manually paginated version of
+    /// this.
+    ///
     /// [Reference](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-playlists-tracks)
+    pub fn playlist_tracks<'a, L: Into<Option<u32>>, O: Into<Option<u32>>>(
+        &'a self,
+        playlist_id: &'a PlaylistId,
+        fields: Option<&'a str>,
+        market: Option<&'a Market>,
+    ) -> impl Paginator<ClientResult<PlaylistItem>> + 'a {
+        paginate(
+            move |limit, offset| {
+                self.playlist_tracks_manual(playlist_id, fields, market, Some(limit), Some(offset))
+            },
+            50,
+        )
+    }
+
+    /// The manually paginated version of [`Spotify::playlist_tracks`].
     #[maybe_async]
-    pub async fn playlist_tracks<L: Into<Option<u32>>, O: Into<Option<u32>>>(
+    pub async fn playlist_tracks_manual<L: Into<Option<u32>>, O: Into<Option<u32>>>(
         &self,
         playlist_id: &PlaylistId,
         fields: Option<&str>,
+        market: Option<&Market>,
```

Comment:
While I was at it I made these parameters consistent; `limit` and `offset` always go last.